pkg/kubelet/util/util.go

```diff
@@ -60,3 +65,50 @@ func GetContainerByIndex(containers []v1.Container, statuses []v1.ContainerStatu
 	}
 	return containers[idx], true
 }
+
+var (
+	tmpfsNoswapOptionSupported        bool
+	tmpfsNoswapOptionAvailabilityOnce sync.Once
+)
+
+const TmpfsNoswapOption = "noswap"
+
+func IsTmpfsNoswapOptionSupported(mounter mount.Interface) bool {
+	isTmpfsNoswapOptionSupportedHelper := func() bool {
+		if sysruntime.GOOS == "windows" {
+			return false
+		}
+
+		mountDir, err := os.MkdirTemp("", "tmpfs-noswap-test")
+		if err != nil {
+			klog.InfoS("error creating dir to test if tmpfs noswap is enabled. Assuming not supported", "mount path", mountDir, "error", err)
```

Comment:
Do we want "info" level here for this error? Perhaps a warning or a real error log level?

Reply:
I was initially using `klog.Warning()`, but the linter complained that I must use structured logging here (i.e. something that ends with an S, `InfoS`, `ErrorS`, etc).

Unfortunately (not sure why) there is no `WarningS`.

But using an `ErrorS` is definitely an option. I chose not to use it since it's not an error in the sense that kubelet cannot start/run properly, but rather a best-effort attempt to use noswap.

Reply:
I see. This is a bit unfortunate and an odd choice not to include this warning print variant.